### PR TITLE
storage/test_storage:fix bug in test_txn_store_gc3

### PR DESCRIPTION
**Pull request by AndreMouche**
Hi,
      This PR fixed bug in `test_txn_store_gc3`, the related issue is https://github.com/pingcap/tikv/issues/1981

@disksing @zhangjinpeng1987 PTAL

### CODE REVIEW

File: tests/storage/test_storage.rs [hunk 1]
```diff
@@ -414,8 +414,8 @@ fn test_txn_store_gc3() {
     let key = "k";
     let store = AssertionStorage::default();
     store.test_txn_store_gc3(key.as_bytes()[0]);
-    let (_cluster, raft_store) = AssertionStorage::new_raft_storage_with_store_count(3, key);
-    raft_store.test_txn_store_gc3(key.as_bytes()[0]);
+    let (mut cluster, mut raft_store) = AssertionStorage::new_raft_storage_with_store_count(3, key);
+    raft_store.test_txn_store_gc3_for_cluster(&mut cluster, key.as_bytes()[0]);
```
Comment (disksing) [inline tests/storage/test_storage.rs]: Can the old func `test_txn_store_gc3` be removed?

Comment (AndreMouche) [inline tests/storage/test_storage.rs]: No, it will still be used on L416 @disksing

**Comment by disksing** [comment]
LGTM.

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**